### DISCUSSION TITLE
2.4.20 bugs

### DISCUSSION THREAD

**Discussion by udi-mosh** [Security-Onion-Solutions/securityonion#11588] [2.4] [answered]
### Version

2.4.20

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

airgap

### Hardware Specs

Does not meet minimum requirements

### CPU

2

### RAM

8

### Storage for /

100

### Storage for /nsm

100

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

i have tried installing v2.4.20 12.10 update on a virtual box. both standalone and distributed failed installing.
i have installed v2.4.20 6.10 and the installation completed. however, i found those bugs:
1. even-though i am the administrator whom installed the system, some settings (elastic defend) can't be configured saying i do not have permissions.
2. i can't delete integrations\policies
3. i can't move agent between policies

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by Acewiza** [comment]
If I though somebody might try to install it under the criteria listed, I'd probably put a few special minimum spec bugs in, too.

**Reply by udi-mosh** [reply]
not sure i understand your comment...

**Comment by TOoSmOotH** [comment, marked as answer]
You would need to do a standalone install to get all the features and your amount of RAM isn't the required minimum specs.
